### ISSUE TITLE
Feature Request: Blacklist for filtering windows by title

### ISSUE THREAD

**Issue by esauvisky** [timbertson/shellshape#197]
Is there a way to make a list to block certain window titles from tiling at all?

There are some programs (including annoying dialog boxes) that tile automatically when you don't want them to.

**Comment by timbertson** [comment]
Nope, there's no support for this currently. Mostly it's a bit of work to store + make the settings UI to manage such a list, there's no technical reason preventing it if someone wanted to attempt it though.
